src/newton.f90

```diff
@@ -97,7 +97,7 @@ subroutine newton( NGdof, position, ihybrd )
 
   INTEGER                :: ML, MU ! required for only Lc05ndf;
 
-  LOGICAL                :: Lexit = .true. ! perhaps this could be made user input;
+  LOGICAL                :: Lexit = .false. ! if false, hessian is printed even with initial force balance
```

Comment:
Does this significantly change the look of the output of most runs? If so, consider keeping it True, otherwise keep false and ignore me.